test/visualTest.ts

```diff
@@ -349,5 +349,83 @@ namespace powerbi.extensibility.visual.test {
                 objectsChecker(jsonData);
             });
         });
+
+        describe("Popup", () => {
```

Comment:
All of tests for this describe must be updated to use Jasmine [spyOn](https://jasmine.github.io/api/edge/global.html#spyOn) instead of using a variable to check if method is called or not.

Reply:
fixed